web-frontend/src/app/globals.css

```
@@ -22,6 +22,7 @@
 :root {
   --background: #ffffff;
   --foreground: #171717;
+  color-scheme: light;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)